packages/frontend/src/translations/en.global.json

```diff
@@ -748,6 +754,10 @@
             }
         }
     },
+    "migration": {
+        "message": "NEAR Wallet has migrated to a new domain: <a href='https://mynearwallet.com' target='_blank'>mynearwallet.com</a>.<br/>We prepared an easy way to move your accounts. Click the button to start.",
```

Comment:
this should be addressed by design already, see [this PR here](https://github.com/near/near-wallet/pull/2752)